LabIFSC/montecarloerror.py

```diff
@@ -0,0 +1,58 @@
+from .medida import Medida
+import pkg_resources
+installed= {pkg.key for pkg in pkg_resources.working_set}
+
+def montecarlo(funcao=sum,N=1e4,parametros=[Medida((2,0.14),"m"),Medida((4,0.1),"m")],hist=False,bins=100,assimetria="false",comparar=False):
+  N=int(N)
+  samples=[]
+  if "numpy" in installed:
+    from numpy.random import normal
+    for j in range(len(parametros)): #numeros aleatorios com numpy
+      samples.append(normal(parametros[j].nominal,parametros[j].incerteza,N))
+  else:
+    from random import gauss
+    for j in range(len(parametros)): #numeros aleatorios com random 
+      samples.append(gauss(parametros[j].nominal,parametros[j].incerteza,N))
+  values=[]
+  for k in range(N):
+    temp=[]
+    for j in range(len(parametros)):
+      temp+=[samples[j][k]]
+    try:
+      values.append(funcao(tuple(temp)).nominal) #caso estejamos usando uma função do LabIFSC
+    except:
+      values.append(funcao(tuple(temp))) #outras funções
+  if "matplotlib" in installed and hist==True:
```

Comment:
if "matplotlib" in installed and hist==True:
    matplotlib = installed['matplotlib']